[ERROR] Cannot read properties of undefined (reading 'url')

In the Schema resolver, when the Api spec contains an empty array of servers, the script fails with

```
{
  "openapi": "3.0.0",
  ...
  "servers": [],
  ...
}
```

`[ERROR] Cannot read properties of undefined (reading 'url')`

https://github.com/povio/openapi-codegen-cli/blob/453c90f906d869bf31670abbcb0f2d4d3e94dbff/src/generators/core/SchemaResolver.class.ts#L252

--- a/package.json
+++ b/package.json
@@ -1,6 +1,6 @@
 {
   "name": "@povio/openapi-codegen-cli",
-  "version": "0.6.4",
+  "version": "0.6.5",
   "main": "./dist/index.js",
   "bin": {
     "openapi-codegen": "./dist/sh.js"

--- a/src/generators/core/SchemaResolver.class.ts
+++ b/src/generators/core/SchemaResolver.class.ts
@@ -249,7 +249,7 @@ export class SchemaResolver {
   }
 
   getBaseUrl() {
-    const serverUrl = this.openApiDoc.servers?.[0].url;
+    const serverUrl = this.openApiDoc.servers?.[0]?.url;
     if (this.options.baseUrl === "" && serverUrl) {
       return serverUrl;
     }

